### DISCUSSION TITLE
评论框过小 || The comment box is too small

### DISCUSSION THREAD

**Discussion by zhoujiegl** [walinejs/waline#2193] [Q&A] [answered]
### Describe the bug | 问题描述

我是基础Jekyll框架的静态网站，然后引入[Waline]之后能显示，但是太小了，很不协调，大家有这种情况吗，怎么解决
![image](https://github.com/walinejs/waline/assets/107822405/9d6d8d13-d9af-4cdf-928e-50210318c78c)

### Website URL | 问题网站

https://zhoujiegl.surge.sh/2023/11/17/demo6/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I have a static website based on the Jekyll framework. After introducing [Waline], it can be displayed, but it is too small and uncoordinated. Has this happened to anyone and how to solve it?
![image](https://github.com/walinejs/waline/assets/107822405/9d6d8d13-d9af-4cdf-928e-50210318c78c)

### Website URL | Problem website

https://zhoujiegl.surge.sh/2023/11/17/demo6/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
```css
html { font-size: 16px; }
```
![image](https://github.com/walinejs/waline/assets/424491/f07efe36-711c-4556-8f31-b892e3b8312b)

**Reply by zhoujiegl** [reply]
可以具体一点怎么修改吗，我代码水平不是很高，拿着模板在改

---
Can you be more specific about how to modify it? My coding level is not very high. I am modifying it with the template
